app/Services/VoucherEvaluator/Evaluations/FamilyHasNoEligibleChildren.php

```diff
@@ -11,7 +11,7 @@
 
 class FamilyHasNoEligibleChildren extends BaseFamilyEvaluation
 {
-    public $reason = 'in need of under ones to qualify primary schoolers';
+    public $reason = 'has no child under primary school age then children of primary school age get no vouchers';
```

Comment:
These are bringing the voucher values into the rule text. 

The danger is that if we do that and circumtances change, then we have to rewrite this

the template might be done as 
"if {entity} {reason} {value} voucher(s)"